test/test_core_notes.py

```diff
@@ -1,47 +1,188 @@
-#------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # elftools tests
 #
 # Maxim Akhmedov (max42@yandex-team.ru)
 # This code is in the public domain
-#------------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 import unittest
 import os
 
 from elftools.elf.elffile import ELFFile
 from elftools.elf.segments import NoteSegment
 
+
 class TestCoreNotes(unittest.TestCase):
     """ This test makes sure than core dump specific
         sections are properly analyzed.
     """
 
-    def test_core_prpsinfo(self):
-        with open(os.path.join('test',
+    def setUp(self):
+        self._core_file = open(os.path.join('test',
```

Comment:
Why the setUp?

Reply:
The opening of the test core file was needed in my test as well as the existing test in the file - it seemed to me that it was a case of duplicate code so I moved that task into a testSetup function given that it is common to both tests.

Browsing through the documentation it seems what I actually want is setUpClass() as there is no need for this to be executed before every test